assets/i18n/en.json

```diff
@@ -136,9 +143,13 @@
     "I18N_BLOG_POST_EDITOR_TAGS_LIMIT_SPECIFICATION_SUFFIX": "more tags can still be added.",
     "I18N_BLOG_POST_EDITOR_THUMBNAIL_HEADING": "Thumbnail",
     "I18N_BLOG_POST_EDITOR_TITLE_HEADING": "Title",
+    "I18N_BLOG_POST_EDITOR_TITLE_MAX_LENGTH_ERROR": "Blog Post title should have at most <[maxChars]> characters.",
+    "I18N_BLOG_POST_EDITOR_TITLE_MIN_LENGTH_ERROR": "Blog Post title should have at least <[minChars]> characters.",
     "I18N_BLOG_POST_PAGE_RECOMMENDATION_SECTON_HEADING": "Suggested For You.",
     "I18N_BLOG_POST_PAGE_TAGS_HEADING": "Tags",
     "I18N_BLOG_POST_PAGE_TITLE": "<[blogPostTitle]> | Blog | Oppia",
+    "I18N_BLOG_POST_THUMBNAIL_ALLOWED_EXTENSIONS": "Allowed image extensions: 'jpeg', 'jpg', 'png', 'svg'",
+    "I18N_BLOG_POST_THUMBNAIL_ALLOWED_MAX_FILE_SIZE": "The maximum allowed image size is 100 KB. Note that, sometimes, cropping might increase the image size.",
```

Comment:
Can we make the file size a constant?

Reply:
Done in https://github.com/oppia/oppia/pull/16299.